Refresh.GameServer/Endpoints/Game/PhotoEndpoints.cs

```diff
@@ -30,6 +30,19 @@ public Response UploadPhoto(RequestContext context, SerializedPhoto body, GameDa
         return OK;
     }
 
+    [GameEndpoint("deletePhoto/{id}", Method.Post)]
+    public Response DeletePhoto(RequestContext context, GameDatabaseContext database, GameUser user, int id)
+    {
+        GamePhoto? photo = database.GetPhotoById(id);
+        if (photo == null) return NotFound;
+
+        if (photo.Publisher.UserId != user.UserId)
+            return Forbidden;
```

Comment:
```suggestion
            return Unauthorized;
```

Sending 403 will cause the game to disconnect and retry the request with a new token, use 401 instead

From MDN:
> The HyperText Transfer Protocol (HTTP) `401 Unauthorized` response status code indicates that the client request has not been completed because it lacks valid authentication credentials for the requested resource.